kong/credentials_service.go

```diff
@@ -109,6 +121,32 @@ func (s *credentialService) Get(ctx context.Context, credType string,
 	return cred, nil
 }
 
+// GetByID fetches a credential of credType with ID from Kong.
+func (s *credentialService) GetByID(ctx context.Context, credType string, credIdentifier *string,
+) (json.RawMessage, error) {
+	if isEmptyString(credIdentifier) {
```

Comment:
Shouldn't we add a similar check for `credType`?

Reply:
We are checking for a valid credType on line 132 - https://github.com/Kong/go-kong/pull/526/files#diff-c7a9e15599f12d918c2a99b837688ce87f76a4a996e369301e424ffa473b24ccR132
Since empty string is a valid key in a map - IMO additional check is not needed